src/main/java/com/artipie/front/rest/BaseService.java

```diff
@@ -262,4 +265,81 @@ protected static JsonObject jsonObject(final HttpResponse<String> response) {
     protected static JsonArray jsonArray(final HttpResponse<String> response) {
         return Json.createReader(new StringReader(response.body())).readArray();
     }
+
+    /**
+     * Convert response json-body to yaml.
+     * @param response Response.
+     * @return Yaml content.
+     */
+    protected static String toYaml(final HttpResponse<String> response) {
+        return new Json2Yaml().apply(BaseService.jsonObject(response).toString())
+            .toString();
+    }
+
+    /**
+     * Strip leading and ending quotes.
+     * @param str String with leading and ending quotes or without them.
+     * @return Stripped string.
+     */
+    protected static String stripQuotes(final String str) {
+        String result = str;
+        if (str.length() > 1 && str.charAt(0) == '"' && str.charAt(str.length() - 1) == '"') {
+            result = str.substring(1, str.length() - 1);
+        }
+        return result;
+    }
+
+    /**
+     * Check response result code and forms triple-tuple:
+     *   1. response status code.
+     *   2. resulting content in case success result code
+     *   3. error message based on response content in case receiveing of error in status code.
+     * @param response Response.
+     * @param map Map-function to form resulting content in case successful status code.
+     * @param <V> Type of resulting of map-function.
+     * @return Triple-tuple.
+     */
+    protected static <V> Tuple3<Integer, V, String> handle(final HttpResponse<String> response,
+        final Function<HttpResponse<String>, V> map) {
+        return handle(HttpServletResponse.SC_OK, response, map);
+    }
+
+    /**
+     * Check response result code and forms triple-tuple:
+     *   1. response status code.
+     *   2. resulting content in case success result code
+     *   3. error message based on response content in case receiveing of error in status code.
+     * @param success Expected success result code.
+     * @param response Response.
+     * @param map Map-function to form resulting content in case successful status code.
+     * @param <V> Type of resulting of map-function.
+     * @return Triple-tuple.
+     */
+    protected static <V> Tuple3<Integer, V, String> handle(final int success,
```

Comment:
There tuples with several items does not look clear and it's not convenient to use them: you should always know what item represents and perform various checks... 
Spark has [error handling](https://sparkjava.com/documentation#error-handling) and exception mappers, maybe we can throw some custom exception on error from Artipie API and handle it with spark? Thus no tuples with nulls and no checks will be required...